api/models/site.js

```diff
@@ -149,6 +149,24 @@ module.exports = (sequelize, DataTypes) => {
       type: DataTypes.STRING,
       allowNull: false,
     },
+    basicAuth: {
```

Comment:
apparently the `getterMethods` section is deprecated. Also, using setters allows the other code to interact with the site model in a more natural way, without needing to worry about these fields actually stored in the `config` field.

Reply:
👍 👍 The virtual data type looks a lot cleaner and definitely makes interacting with `config` more natural.